qa/workunits/suites/mgr_mod_snap_schedule.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/bash
+TESTDIR=$(dirname $0)
+
+echo "TESTDIR:$TESTDIR"
+
+ceph mgr module enable snap_schedule
+sleep 2
+ceph config set mgr mgr/snap_schedule/allow_m_granularity True
+sleep 2
+mkdir $CEPH_MNT/t
+ceph fs snap-schedule add /t 1M
+pushd $CEPH_MNT/t
+
+(python -c "import os
+from datetime import time
+
+fd = os.open('./data-file', os.O_WRONLY | os.O_CREAT, 0o777)
+assert fd is not None'
+
+start = time.time()
+data = 'A'
+while time.time() - start < 1140.0:
+    str = data * 1024
+    os.write(fd, str)
+    next = ord(data) - ord('A') + 1
+    next %= 26
+    data = chr(next + ord('A'))
+    time.sleep(13)
+
+os.close(fd)
+") &
+
+let -i start=$(date +%s)
+while true;
+do
+  let -i now=$(date +%s)
+  let -i elapsed=$((now - start))
+  if [ $elapsed -ge 1140 ]; then
+    break
+  fi
+  ceph fs snap-schedule status /t || echo "$0: snap-schedule status command failed with status $?"
+  sleep 1
+done
+
+ceph fs snap-schedule deactivate /t 1M
+sleep 60
+
+echo 'cephfs:'; ls .snap
+let -i count=$(ls .snap | wc -l)
+find .snap -maxdepth 1 -mindepth 1 -type d -exec rmdir {} ';'
+popd
+rmdir $CEPH_MNT/t
+[ $count -ge 15 ] && true || false
```

Comment:
All such magic numbers are best to be declared somewhere so that they can be tweaked at once place whenever necessary.

Reply:
magic numbers have been placed at the top of the same script